docs/html/reference/pip_install.rst

```diff
@@ -148,6 +148,8 @@ and the newline following it is effectively ignored.
 
 Comments are stripped *before* line continuations are processed.
 
+To interpret the requirements file in UTF-8 format add a comment ``# -*- coding: utf-8 -*-`` to the first or second line of the file.
```

Comment:
I would break this line at column 80 or so (without splitting the monospaced text), so it is not as long when viewed as plain text.